ArduPlane/servos.cpp

```diff
@@ -104,7 +104,7 @@ bool Plane::suppress_throttle(void)
         return false;
     }
 
-    bool gps_movement = (gps.status() >= AP_GPS::GPS_OK_FIX_2D && gps.ground_speed() >= 5);
+    bool gps_movement = (gps.status() >= AP_GPS::GPS_OK_FIX_2D && gps.ground_speed() >= GPS_GND_CRS_MIN_SPD);
```

Comment:
This is not quite the same use case, I don't think we should change to the new define.

Reply:
okay